### ISSUE TITLE
4.4.4 Release Tasks

### ISSUE THREAD

**Issue by geerlingguy** [geerlingguy/drupal-vm#1330]
Working on the 4.4.4 release...

  - [x] Update CHANGELOG
  - [x] Cut a <s>rug</s> tag
  - [x] Add tag release notes
  - [x] <s>Submit PR to update BLT</s> (not needed for this one)
  - [x] <s>Blog post on Planet</s> (not needed for this one)

**Comment by geerlingguy** [comment]
Just waiting on this build to finish: https://travis-ci.org/geerlingguy/drupal-vm/builds/224772778

**Comment by geerlingguy** [comment]
(Though it will likely be time for dinner by the time it finishes, so I'll tag and release later today.)

**Comment by geerlingguy** [comment]
Complete: https://github.com/geerlingguy/drupal-vm/releases/tag/4.4.4
